libraries/AP_Mission/AP_Mission_Commands.cpp

```diff
@@ -165,6 +165,17 @@ bool AP_Mission::start_command_camera(const AP_Mission::Mission_Command& cmd)
         }
         return false;
 
+    case MAV_CMD_SET_CAMERA_SOURCE:
+        if (cmd.content.set_camera_source.instance == 0) {
+            // set lens for every backend
+            bool ret = false;
+            for (uint8_t i=0; i<AP_CAMERA_MAX_INSTANCES; i++) {
```

Comment:
ARRAY_SIZE preferred

Reply:
@tridge, thanks for the review.  From within the AP_Mission library we don't have access to the backends array so I don't see how we could use an ArraySize.  The AP_Camera interface already has protection against invalid instance arguments being passed in so I think I can leave this as-is.